data/movements/scripts/walkback.lua

```diff
@@ -1,6 +1,6 @@
 function onStepIn(creature, item, position, fromPosition)
 	if item.uid > 0 and item.uid <= 65535 then
-		creature:teleportTo(fromPosition, false)
+		return false
 	end
 	return true
```

Comment:
uid is over 65535 for items without a unique id, this is how we used to reference items before item userdata.